examples/js/exporters/GLTFExporter.js

```diff
@@ -1088,13 +1088,15 @@ THREE.GLTFExporter.prototype = {
 			}
 
 			// alphaMode
-			if ( material.transparent || material.alphaTest > 0.0 ) {
+			if ( material.transparent ) {
 
-				gltfMaterial.alphaMode = material.opacity < 1.0 ? 'BLEND' : 'MASK';
+				gltfMaterial.alphaMode = 'BLEND';
 
-				// Write alphaCutoff if it's non-zero and different from the default (0.5).
-				if ( material.alphaTest > 0.0 && material.alphaTest !== 0.5 ) {
+			} else {
+
+				if ( material.alphaTest > 0.0 ) {
```

Comment:
Probably just want an `else if` here?

Reply:
Kind of reads better to me like this: "If not transparent, then... If alphaTest..." 🤪

Originally I wrote it like that because I thought I would have to set `gltfMaterial.alphaMode = 'OPAQUE';` but I guess it the default.

I think I'll leave it like this for now.